wizeline/wize-q-front
# Feat - WIP 192 - Backend search

#### What does this PR do?
- Incorporates backend search functionality. 
- Removes frontend search functionality.
#### Where should the reviewer start?
- Verify action creator in index.js file. fetchQuestions function has been
modified to receive a new parameter named 'search'.
- Verify Questions Reducer. It nows take into consideration the search term to
append results or replace them.
#### How should this be manually tested?
- Log in to the application.
- Type a term in 'search' field
- Verify search results update according your search term.
- Delete some characters from the field to validate that results update as well.
#### Any background context you want to provide?
Search functionality was already implemented in a partial way. 
(A front end filter was being triggered). Now, the user is able to search in 
all the saved questions and not only in the displayed ones.
#### What are the relevant tickets?
[WIP-192 Search/Filter](https://wizeline.atlassian.net/browse/WIP-192)
#### Screenshots (if appropriate)
<img width="1153" alt="screen shot 2017-06-15 at 12 03 45 pm" src="https://user-images.githubusercontent.com/2502032/27193106-4ee591be-51c3-11e7-9816-ba0a33d3adaf.png">
